tests/integration_tests/__init__.py

```diff
@@ -0,0 +1,4 @@
+import unittest
+
+# Hides first line of function docstring in test output
+unittest.TestCase.shortDescription = lambda x: '\t'
```

Comment:
Is this for the `"""\n` part? I've noticed almost all our docstrings to start with an empty line and it's become a somewhat unspoken convention. I think this will help standardize that convention.

Reply:
Thanks, I see the intent now. It is odd, but the tab doesn't seem to solve it, rather pushes things to a newline. I think the `lambda x: None` is good.